Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -323,7 +323,7 @@ def audit_file
       end
       valid_alias_names = [alias_name_major, alias_name_major_minor]
 
-      if formula.tap
+      unless ["homebrew/core", ""].include? formula.tap.to_s
```

Comment:
Ahh well, I guess no esoteric/eccentric styling that doesn't match that of the rest of the code. 😆